Week3/homework/step2-2.js

```diff
@@ -3,18 +3,28 @@
 function threeFive(startIndex, stopIndex, threeCallback, fiveCallback) {
   const numbers = [];
 
-  // Replace this comment and the next line with your code
-  console.log(startIndex, stopIndex, threeCallback, fiveCallback, numbers);
+  for (let i = 0; i <= stopIndex - startIndex; i++) {
+    numbers.push(startIndex + i);
+  }
+
+  for (const number of numbers) {
+    if (!(number % 3)) {
+      threeCallback(number);
+      if (!(number % 5)) {
+        fiveCallback(number);
+      }
+    } else if (!(number % 5)) {
+      fiveCallback(number);
+    }
+  }
 }
```

Comment:
Instead of using an `if-then-else` statement, two simple, independent `if` statements are all that is needed here.

```js
for (const number of numbers) {
  if (number % 3 === 0) {
    threeCallback(number);
  }
  if (number % 5 === 0) {
    fiveCallback(number);
  }
}
```